src/pages/History/Filters/index.tsx

```diff
@@ -0,0 +1,63 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import { Paper, Typography } from '@mui/material';
+import { Checkbox, Select } from 'components';
+import { useTranslation } from 'translation';
+import { TransactionEvent } from 'types';
+import { useStyles } from './styles';
+
+export const AllValue = 'All';
```

Comment:
That's a minor thing, but since it's a constant: 
```suggestion
export const ALL_VALUE = 'All';
```